src/api/clinical/diagnosis/diagnosis.controller.ts

```diff
@@ -91,13 +91,12 @@ export class DiagnosisController extends BaseController{
         try {
 
             const domainModel = await this._validator.update(request);
-
             const id: uuid = await this._validator.getParamUuid(request, 'id');
```

Comment:
This needs to be looked at. There are some problems.

Reply:
Done.